Drop-In/src/main/res/layout/bt_drop_in_activity.xml

```diff
@@ -55,17 +55,36 @@
                     android:animateLayoutChanges="true"
                     android:visibility="gone">
 
-                    <TextView
-                        style="@style/bt_choose_payment_label"
-                        android:paddingTop="@dimen/bt_choose_existing_payment_label_padding_top"
-                        android:text="@string/bt_recent"/>
-
-                        <android.support.v7.widget.RecyclerView
-                            android:id="@+id/bt_vaulted_payment_methods"
-                            android:layout_width="match_parent"
+                    <RelativeLayout
+                        android:orientation="horizontal"
+                        android:layout_width="wrap_content"
+                        android:layout_height="wrap_content">
+
+                        <TextView
+                            style="@style/bt_choose_payment_label"
+                            android:paddingTop="@dimen/bt_choose_existing_payment_label_padding_top"
+                            android:text="@string/bt_recent"/>
+
+                        <Button
+                            style="@style/Widget.AppCompat.Button.Borderless.Colored"
+                            android:theme="@style/bt_edit_button"
+                            android:id="@+id/bt_vault_edit_button"
+                            android:layout_width="wrap_content"
                             android:layout_height="wrap_content"
-                            android:paddingLeft="4dp"
-                            android:paddingRight="4dp"/>
+                            android:paddingTop="@dimen/bt_choose_existing_payment_label_padding_top"
```

Comment:
I think this is throwing off the centering of the text within the touch target which makes the touch ripple look kinda off